docs/source/integrations/marimo.md

```diff
@@ -0,0 +1,36 @@
+# How to use Kedro from Marimo and non-Jupyter environments
+
+This guide explains how to set up Kedro programmatically without relying on IPython magics, making it compatible with non-Jupyter environments (python scripts or streamlit apps) and modern notebook interfaces like Marimo.
+
+## Overview
+
+Kedro Jupyter notebook users rely on the magic command `%load_ext kedro.ipython` to source their catalog, context, pipelines and session objects, see [Kedro and Notebooks](../notebooks_and_ipython/kedro_and_notebooks.md) for more. This magic command automatically finds the location of the root kedro project, avoiding having users hardcode paths to their configuration files.
+
+Such magic commands are not available in modern notebook interfaces such as marimo or in simple python scripts. Yet, using Kedro for sharing configuration, credentials or the data catalog across a project is still desired when developing data science projects.
```

Comment:
While factually correct, I think I'd focus this page on describing how to run Kedro on marimo, so trim down this introduction a bit. Some other possible additions:

- Do not assume that the reader knows marimo, hence include a sentence linking to its official website and explaining briefly what it is
- Add the necessary `marimo edit` or `marimo run` commands
- Add a screenshot that demonstrates how this looks like

Reply:
Would you want to focus solely on marimo? I think it could be relevant to talk about non jupyter environments as well: streamlit, python scripts or even cli scripts

Reply:
and also, what is the screenshot meant to show? I think what I might do is create a start that uses marimo, I'd need a screenshot of a starter project that's not my work projects anyway